nevergrad/benchmark/experiments.py

```diff
@@ -115,6 +117,22 @@ def mltuning(
     optims: tp.List[str] = get_optimizers("basics", seed=next(seedg))  # type: ignore
     if not seq:
         optims = get_optimizers("oneshot", seed=next(seedg))  # type: ignore
+    optims += [
+        "Shiwa",
+        "NGO",
+        "NGOpt4",
+        "NGOpt8",
+        "NGOpt10",
+        "NGOpt12",
+        "NGOpt13",
+        "NGOpt14",
+        "NGOpt15",
+        "NGOpt16",
+        "NGOpt21",
+        "NGOpt36",
+        "NGOpt38",
+        "NGOpt",
+    ]
```

Comment:
use an optimizerlist, why not "ngopts" directly? you can also have a reduced ngopts